src/autoscaler/eventgenerator/aggregator/aggregator_test.go

```diff
@@ -232,16 +234,17 @@ var _ = Describe("Aggregator", func() {
 	})
 	Context("Start", func() {
 		JustBeforeEach(func() {
-			aggregator = NewAggregator(logger, clock, testPolicyPollerInterval, policyDatabase, appMetricDatabase, metricServer.URL(), metricPollerCount, evaluationManager, appMonitorChan)
+			aggregator = NewAggregator(logger, clock, testAggregatorExecuteInterval, testPolicyPollerInterval, policyDatabase, appMetricDatabase, metricServer.URL(), metricPollerCount, evaluationManager, appMonitorChan)
 			aggregator.Start()
+			Eventually(clock.WatcherCount).Should(Equal(2)) //policyPoller:1,aggregator:1
```

Comment:
Above when testing ConsumePolicy we need ensure the appMonitors and triggers which are parsed from policies to be both checked. So aggregator and appEvaluationManager should both be started and the ticker count here is 3(aggregator,policyPoller and appEvaluationManager). And for here we test aggregator should be started. So I think the appEvaluationManager need not be started though it can be started. So the ticker count should be 2. I updated it to 3 for consistence.